src/renderer/library/components/dialog/Information.tsx

```diff
@@ -5,22 +5,27 @@
 // that can be found in the LICENSE file exposed on Github (readium) in the project repository.
 // ==LICENSE-END==
 
-import * as commonmark from "commonmark";
-import { readFile } from "fs";
+import { existsSync, promises } from "fs";
 import * as path from "path";
+import { TaJsonSerialize } from "r2-lcp-js/dist/es6-es2015/src/serializable";
+import { Link } from "r2-shared-js/dist/es6-es2015/src/models/publication-link";
```

Comment:
`from "r2-shared-js/dist/es6-es2015/src/models/publication-link`
=>
`from "@r2-shared-js/models/publication-link`

Same with other instances of `"r2-.*-js/` (regexp for VSCode global search)

Reply:
(remember to execute the command "TSI: Sort Imports" afterwards :)

Reply:
fixes 20babcc9